packages/lib-utils/src/components/status/StatusBox.tsx

```diff
@@ -0,0 +1,72 @@
+// @ts-nocheck
+import * as React from 'react';
+import { Alert, Bullseye, Spinner, Text, Title } from '@patternfly/react-core';
+import * as _ from 'lodash';
+
+type AccessDeniedProps = {
+  message?: string;
```

Comment:
Wondering if we should have a fallback message if none are provided? Or we could hide the error details section if no message is provided. I'm thinking it would look odd to see `Error details` in the UI and underneath is blank :)

Reply:
Absolutely, I'll add it.